types/global.d.ts

```diff
@@ -56,3 +56,14 @@ export interface HandlerMap {
 }
 
 export type HandlerType = keyof HandlerMap;
+
+export interface Option {
```

Comment:
I'm a little surprised at taking out `[key: string]: unknown;` from this type.  See the extra fields in `ui/raidboss/raidboss_options.js`.  `raidboss_config.js` also sets a lot of options as well.

Is this intended to be `BaseOptions`, e.g. the interface that something like `RaidbossOptions` would derive from?